content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -159,10 +123,17 @@ export class _CollapsibleSection extends React.PureComponent {
         data-section-id={id}>
         <div className="section-top-bar">
           <h3 className="section-title">
-            <span className="click-target" onClick={this.onHeaderClick}>
-              {this.renderIcon()}
-              {getFormattedMessage(title)}
-              {isCollapsible && <span className={`collapsible-arrow icon ${collapsed ? "icon-arrowhead-forward-small" : "icon-arrowhead-down-small"}`} />}
+            <span className="click-target-container">
+              <span className="click-target" onClick={this.onHeaderClick}>
+                {this.renderIcon()}
+                {getFormattedMessage(title)}
```

Comment:
Why not just use `<FormattedMessage id={title} />` here?

Reply:
Agreed, I can update it. The why is simply I moved it and didn't think much of it ;)

Reply:
I think `getFormattedMessage` is intended. It looks like it does checks to ensure if title is a string, it just uses the string, if it's an object, it uses `FormattedMessage` and I'm not yet sure about why title can sometimes be a string and sometimes be an object, but I'm guessing we should leave it as is.